packages/utils/crates/utils-types/src/primitives.rs

```diff
@@ -500,6 +500,12 @@ pub struct U256 {
 }
 
 impl U256 {
+    pub fn from_big_endian(bytes: &[u8]) -> Self {
+        Self {
+            value: u256::from_be_bytes(bytes.try_into().expect("invalid input size"))
+        }
+    }
+
```

Comment:
A `From` returning result would be a better solution in order not to break the runtime with panic.